docs/source/tutorials/use.md

```diff
@@ -41,3 +41,17 @@ For developments and debugging, you can run the package directly from python sou
 conda activate lidar_prod
 python lidar_prod/run.py paths.src_las=[/path/to/file.las]
 ```
+## Run Different tasks
```

Comment:
Pour la documentation, il manque un passage dans le description des processus : 
https://ignf.github.io/lidar-prod-quality-control/background/production_process.html

On peut imaginer l'ordre suivant : (A) Identification of Vegetation Points (B) Identification of Unclassified Points, (C) Building Points, avec C.1: Building Validation, C.2 Building Completion, C.3 Building Identification

C'est une qui sera partagée en réponse à la question "Que fait votre outil ?".

Reply:
Pour l'instant les détections de la végétation et des non classés ne sont pas dissociables par l'utilisateur. Y aurait-il une raison qu'elles le soient ?
Si on reste comme cela ce serait alors plutôt :
A.1 végétation
A.2 Non classés
B.1 building validation
B.2 Building Completion
B.3 Building Identification

Pour correspondre à ton sommaire il faudrait dissocier les deux opérations et dire de les faire l'une après l'autre (avec le cout d'ouvrir et de sauvegarder deux fois le las), mais il peut y avoir une raison

Reply:
FInalement j'ai séparé par tâche (le plan que j'ai indiqué), cela me semble plus logique car l'utilisateur veut savoir comment accomplir les tâches, ce qui se passe derrière le rideau l'intéresse moins